core/src/test/scala/org/apache/spark/io/ZStandardBenchmark.scala

```diff
@@ -101,4 +102,28 @@ object ZStandardBenchmark extends BenchmarkBase {
       }
     }
   }
+
+  private def parallelCompressionBenchmark(): Unit = {
+    val N = 128
```

Comment:
If you don't mind, please don't override this.

Reply:
Do you mean renaming it？

Reply:
Oh, no. I thought we can use the following instead of this.
https://github.com/apache/spark/blob/1d80d80a56c418f841e282ad753fad6671c3baae/core/src/test/scala/org/apache/spark/io/ZStandardBenchmark.scala#L39

Reply:
Never mind. I saw the new variable, `val numberOfLargeObjectToWrite = 128`.